collection-scripts/gather

```diff
@@ -33,11 +33,13 @@ resources+=(imagecontentsourcepolicies.operator.openshift.io)
 resources+=(networks.operator.openshift.io)
 
 # Run the Collection of Resources using inspect
-# running accross all-namespaces for the few "Autoscaler" resouces.
 for resource in "${resources[@]}"; do
     oc adm inspect --dest-dir must-gather ${resource}
 done
 
+# Gather optional operator resources from all namespaces
```

Comment:
IMO this should go into its own script (encase we ever need to add/filter things). Calling it by default from this script is fine (provided the collection is not astronomical amounts of data/information).